Maestro/subscriptions.json

```diff
@@ -69,7 +69,55 @@
             "-GitHubEmail dotnet-bot@microsoft.com",
             "-GitHubPassword `$(`$Secrets['DotNetBotGitHubPassword'])",
             "-CoreFxVersionUrl https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/master/Latest.txt",
-            "-GitHubPullRequestNotifications 'dotnet/corefx-contrib'"
+            "-GitHubPullRequestNotifications 'dotnet/corefx-contrib",
+            "-DirPropsVersionElement 'CoreFxExpectedPrerelease"
+          ]
+        }
+      ]
+    },
+    {
+      "path": "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectk-tfs/master/Latest.txt",
+      "handlers": [
+        // This handler will bring the Latest ProjectK TFS master build into CoreFX master for CoreCLR dependencies
+        {
+          "maestroAction": "corefx-general",
+          "maestroDelay": "00:10:00",
+          "ScriptFileName": "UpdateDependencies.ps1",
+          "Arguments": [
+            "-GitHubUser dotnet-bot",
+            "-GitHubEmail dotnet-bot@microsoft.com",
+            "-GitHubPassword `$(`$Secrets['DotNetBotGitHubPassword'])",
+            "-CoreFxVersionUrl https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/projectk-tfs/master/Latest.txt",
```

Comment:
This looks like it is going to cause 3 PR's to corefx whenever we update projectk-tfs. I suspect you are just setting them up independently since we are going they will be independent soon.